docs/source/quickstart.rst

```diff
@@ -100,11 +108,20 @@ Run the command below to download the results
 Clean up
 --------
 This step is **critical**, please do not omit it, even if you ran Ctrl-C when
-starting ElasticBLAST. It is also recommended each time you start a new
+starting ElasticBLAST. If your cluster stays up, you will accrue charges from
+your cloud provider.  It is also recommended each time you start a new
 ElasticBLAST search. 
 
 .. code-block:: bash
 
     ./elastic-blast delete --cfg ${CONFIG_FILE} --loglevel DEBUG
 
 
+The delete command will take a few minutes to run as it needs to shut the cluster down.
+You may verify that your cluster has been deleted by running: 
+
+.. code-block:: bash
+
+  gcloud container clusters list 
+
+This will show all clusters running in your project (even from other users).  If nothing is returned, then no clusters are running.
```

Comment:
Should we also talk about persistent disk here and provide command lines to manually delete both?

Reply:
This is fine, but I'm going to start a new round of changes on Wednesday (aiming to be done by 8/14), and I'd prefer to leave it until then.